governance/articles-of-association.md

```diff
@@ -0,0 +1,52 @@
+# Articles of Association for Pulumiverse
+
+Pulumiverse is a non-profit organization to grow a community to share ideas,
+projects and build Open Source software amongst Pulumi users, developers and
+enthusiasts.
+
+This document regulates this organization; you'll find information about
+becoming a contributor, how and when to vote for board members and how policies
+are changed.
+
+## Contributors
+
+Contributors are the core of Pulumiverse. To become a contributor, all you have
+to do is open an issue(s) or pull request(s). Contributors can earn voting
+rights by either becoming a member of the Open Source Initiative or submitting
+more than ten contributions collectively across Pulumiverse repositories. Voting
+rights are only made valid by being added to the membership roster (feel free to
+open a pull request).
```

Comment:
Not sure how we solve a membership roster, ideas welcome. Maybe we don't need it?